projects/components/src/filtering/filter/parser/types/comparison-filter-parser.ts

```diff
@@ -0,0 +1,62 @@
+import { assertUnreachable } from '@hypertrace/common';
+import { isBoolean } from 'lodash-es';
+import { FilterAttribute } from '../../filter-attribute';
+import { FilterAttributeType } from '../../filter-attribute-type';
+import { FilterOperator } from '../../filter-operators';
+import { FilterParser } from '../filter-parser.decorator';
+import { AbstractFilterParser } from './abstract-filter-parser';
+
+@FilterParser({
+  supportedAttributeTypes: [FilterAttributeType.Boolean, FilterAttributeType.Number, FilterAttributeType.String],
```

Comment:
How do we match parsers? Not all of these attribute types work with all of these operators.

Reply:
Actually going through that part of the code now as I build the lookup service. My thought at the moment is to do it by operator.

Reply:
hmm - we can merge this as is, but may have questions once we get to the lookup part. Curious how the static values are used.